src/index.ts

```diff
@@ -8,11 +8,11 @@ import validateRequestPlugin from './plugins/validate-request'
 
 const Plugins = [
   noticePlugin,
-  authPlugin,
   paginationPlugin,
   registerEndpointsPlugin,
   registerApiEndpointsPlugin,
   validateRequestPlugin,
+  authPlugin,
```

Comment:
Any particular reason for changing the order?

Reply:
I will move it back in the next commit.